Assets/Scripts/UIManager.cs

```diff
@@ -49,37 +50,34 @@ void Update()
     {
         if (GameOver)
         {
-            if (Input.GetKeyDown(KeyCode.R))
+            if (Keyboard.current.rKey.wasPressedThisFrame)

```

Comment:
Werkt dit op controller?

Reply:
zo inderdaad niet, maar in commit 2 is dit algemener gemaakt.